oio/container_streaming/app.py

```diff
@@ -563,17 +562,17 @@ def read(size):
                 if inf.name == CONTAINER_PROPERTIES:
                     assert not hdrs, "invalid sequence in TAR"
                     hdrs = json.loads(read(inf.size))
-                    self.conn.container_update(account, container, hdrs)
+                    self.proxy.container_update(account, container, hdrs)
```

Comment:
Done
We could emit a info log when using deprecated stuff (for developers only ?)